### ISSUE TITLE
Documentation feature request: using "ignore" directive with line continuations

### ISSUE THREAD

**Issue by jasonkarns** [koalaman/shellcheck#3227]
Could someone add an example for how the ignore directive should be used with line continuations?

Say:

```bash
          # shellcheck disable=SC2086
          gh release view $tag || \
            gh release create ${tag/*-*/"$tag" --prerelease} --generate-notes
```

The directive in this case should apply to the gh-release-create line.

Also, for extra confusion, the issue in question is being caught by the github-actions linter in SuperLinter. Which means this is linting the shell scripts that are embedded in github actions workflow yaml, (not a plain shell script file). So any suggestions for "whole-file" directives aren't really helpful here (unless each shell block in the yaml file is treated like a separate file?)

**Comment by marveloo** [comment]
I was unable to reproduce the issue. This shellcheck directive successfully ignored both of the `$tag`s.

**Comment by jasonkarns** [comment]
The issue is that it's not documented. I'm requesting that shellcheck's "next line" ignore syntax document how it is expected to behave with line continuations.

The example above was taken out of a github yaml file and was still triggering an error when linted via github's superlinter. (which does who knows what kind of pre-processing of the yaml file before sending the shell properties to shellcheck) So it's not something that was easily testable locally within vim or from cli with shellcheck.

**Comment by marveloo** [comment]
Depending on the context I would guess your backslash needs escaping with an additional backslash.

**Comment by wileyhy** [comment]
OP's example:

  <sh>
  # shellcheck disable=SC2086
  gh release view $tag || \
      gh release create ${tag/*-*/"$tag" --prerelease} --generate-notes
  <\sh>

On shellcheck.net, in sh and dash the only issues detected from this
version of the script, below, are SC2060 (String replacement...). All five
shell crash bangs, however, pass in terms of the (amended) line
continuation syntax between lines 4 and 5.

  <sh>
  #!/bin/SHELL
  tag=1
  # shellcheck disable=SC2086
  gh release view $tag ||
    gh release create ${tag/*-*/"$tag" --prerelease} --generate-notes
  <\sh>

**Comment by jasonkarns** [comment]
Since it seems this might be limited to superlinter, I'm just going to close this. Thanks all.
